src/Components/CookieNotice.js

```diff
@@ -73,6 +74,7 @@ type Props = {|
   readMoreButtonLink?: string,
   openInNewTab?: boolean,
   cookieTextLabel?: string,
+  cookieName?: string,
```

Comment:
Can you place this line between `cookieExpiration` prop and `darkTheme` prop please ?